docs/developer/apis/camino-node-apis/p-chain.md

```diff
@@ -938,6 +938,179 @@ curl -X POST --data '{
+platform.getClaimables(
+    {
+        locktime: uint64
+        threshold: uint32
+        addresses: []string
+        depositTxIDs: []string
+    }
+
+) -> {
+    depositRewards: uint64,
+    validatorRewards: uint64,
+    expiredDepositRewards: uint64,
+}
+```
+
+- `locktime`: locktime of the rewards
+- `threshold`: the required threshold of
+- `addresses`: list of addresses of the reward Owner
+- `depositTxIDs`: transaction addresses for the deposits
+- `depositRewards`: deposit rewards in nCAMs,
+- `validatorRewards`: validator rewards in nCAMs,
+- `expiredDepositRewards`: expired deposit rewards in nCAMs,
+
+**Example Call**
+
+```sh
+curl -X POST --data '{
+    "jsonrpc": "2.0",
+    "method": "platform.getClaimables",
+    "params": {
+        "locktime": 0,
+        "threshold": 1,
+        "addresses": [
+            "P-columbus1m8wnvtqvthsxxlrrsu3f43kf9wgch5tyfx4nmf"
+        ],
+        "depositTxIDs": [
+            "2Eug3Y6j1yD745y5bQ9bFCf5nvU2qT1eB53GSpD15EkGUfu8xh"
+        ]
+    },
+    "id": 1
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/P
+```
+
+**Example Response**
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "depositRewards": 0,
+    "validatorRewards": 0,
+    "expiredDepositRewards": 0
+  },
+  "id": 1
+}
+```
+
+### platform&#46;getConfiguration
+
+Returns PlatformVM Configuration
+
+**Signature**
+
+```sh
+platform.getConfiguration() -> {
+    networkID: int,
+    assetID: string,
+    assetSymbol: string,
+    hrp: string,
+    blockchains[]{
+        id: string,
+        name: string,
+        subnetID: string,
+        vmID: string
+      },
+    minStakeDuration: uint64,
+    maxStakeDuration: uint64,
+    minValidatorStake: uint64,
+    maxValidatorStake: uint64,
+    minDelegationFee: uint32,
+    minDelegatorStake: uint64,
+    minConsumptionRate: uint64,
+    maxConsumptionRate: uint64,
+    supplyCap: uint64,
+    codecVersion: uint16,
+    verifyNodeSignature: bool,
+    lockModeBondDeposit: bool
+}
+```
+
+- `networkID`: the ID of the Network
+- `assetID`: the fee assed ID
+- `assetSymbol`: the symbol of the fee asset ID
+- `hrp`: `beech32HRP` used in the addresses
+- `blockchains[]`: a list of Primary network blockchains
+- `minStakeDuration`: The minimum duration a validator has to stake
+- `maxStakeDuration`: The maximum duration a validator can stake
+- `minValidatorStake`: The minimum amount of tokens one must bond to be a validator
+- `maxValidatorStake`: The maximum amount of tokens bondable to a validator
+- `minDelegationFee`: The minimum delegation fee
+- `minDelegatorStake`: Minimum stake, in nCAM, that can be delegated on the primary network
+- `minConsumptionRate`: The minimum consumption rate
+- `maxConsumptionRate`: The maximum consumption rate
+- `supplyCap`: The supply cap for the native token (CAM)
+- `codecVersion`: The codec version used for serializing
+- `verifyNodeSignature`: Camino VerifyNodeSignature
+- `lockModeBondDeposit`: Camino LockModeBondDeposit
```

Comment:
@peak3d probably, this struct should also include validator reward import period and dao bond amount